### PR TITLE
doc: Record the steps used to reset the server database

### PR DESCRIPTION
**Pull request by gwhitney**
I have run exactly the steps added in server-administration.md on the
  production machine (they were recorded contemporaneously as I was doing
  them). Production is now serving the offset-test sequences with the
  correct offsets.

### CODE REVIEW

**Review by katestange — APPROVED** [review]
Looks great, one small comment you can ignore if you like.

File: doc/server-administration.md [hunk 1]
```diff
@@ -86,12 +86,46 @@ thinks it is running correctly, and you can point your browser for example at
...
+
+5. Remove the existing database with `sudo -u scope dropdb [NAME]`.
+
+6. Remove the generated database structure with
+   `sudo -u scope rm -r migrations`.
+
+7. Initialize a brand new database with `sudo -u scope createdb [NAME]`.
+
+7. Remove old log files (make sure that there is nothing in them you may need;
+   if so, move them to some out-of-the-way location rather than deleting them)
```
[the remark this answers is not in the document]
Comment (gwhitney) [reply doc/server-administration.md]: Yes, that item was awkward. Have totally re-worded, and also fixed the double occurrence of item 7 in that list. Hopefully all is well now for merging.

**Review by katestange — APPROVED** [review]
Oh I didn't even notice the double 7's :)

**Comment by gwhitney** [comment]
OK, this and #124 are installed and running in production, bringing us up-to-date at the moment.

**Pull request merged**